src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -280,11 +280,15 @@ export default {
 
 </script>
 
-<style scoped>
+<style lang="postcss" scoped>
 .amountDropdownWrapper >>> select {
 	border-radius: 14px 0 0 14px;
 }
 
+.lend-again >>> span {
+	@apply tw-px-0.5 md:tw-px-1;
```

Comment:
I'm specifically seeing this padding getting added to both child `span` elements of the button, so the two paddings are adding, pushing the content. I solved this locally by changing this to:

```
.lend-again ::v-deep span {
	@apply tw-px-0;
}

.lend-again ::v-deep > span {
	@apply tw-px-1;
}
```

Reply:
oh, looks like this is fixing too with no extra lines

```
.lend-again >>> span:first-child {
	@apply tw-px-0.5 md:tw-px-1;
}
```

<img width="360" alt="Screenshot 2023-03-01 at 16 16 00" src="https://user-images.githubusercontent.com/94026278/222277978-6a5340d7-d449-479f-b52f-8a2f378a9248.png">

Reply:
Interesting, I thought both `span` elements were first children.

Reply:
fixed

Reply:
Still seeing the issue locally:

![image](https://user-images.githubusercontent.com/16867161/222280151-6d6a8ac2-cc47-41c2-bc1b-a71bd3441939.png)

Reply:
Oh I underlined the wrong selector, but you can see `first-child` applying the padding to this second `span`. The first-child is applied per parent element.

Reply:
Nice, even I was testing in LH page, lemme see with your suggestion above